curriculum/challenges/english/blocks/workshop-shortest-path-algorithm/686253f1b051998ad4904e3e.md

```diff
@@ -31,7 +31,7 @@ The function should take in three parameters: `matrix`, `start_node`, and `targe
 ```js
 ({
   test: () => {
-    assert(runPython(`_Node(_code).find_function("shortest_path").has_args(["matrix", "start_node", "target_node"])`))
+    assert(runPython(`_Node(_code).find_function("shortest_path").has_args(["matrix", "start_node", "target_node=None"])`))
```

Comment:
`has_args` take a string, not a list, please see the docs https://contribute.freecodecamp.org/curriculum-help/#has_args

Reply:
just say in the hint that `target_node` should have a default value here instead of adding the next test

Reply:
haven't pushed the changes, still working on other feedback